crates/graphql/src/schema/query_root.rs

```diff
@@ -540,4 +541,43 @@ impl QueryRoot {
     fn denylist() -> Denylist {
         Denylist
     }
+
+    #[graphql(description = "Global summary statistics over indexed transactions.")]
+    fn stats_global(&self, context: &AppContext) -> FieldResult<GlobalStats> {
+        let conn = context.shared.db.get().context("failed to connect to db")?;
+
+        let i: u32 = 1000;
```

Comment:
```suggestion
        let i = 1000_u32;
```
Can we have a more descriptive variable name here?